### DISCUSSION TITLE
[Feature] Add search context direct from search bar (Date, Face Name, Location)

### DISCUSSION THREAD

**Discussion by iansewell** [immich-app/immich#8769] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Would be great to have some shortcuts for the new improved search context.

Some ideas:

- if a date is entered. (1/1/2023 or Jan 1 2023) then translate that to a search context and filter search results to that day
- if a face name is entered then add that face to the context.  Might need to prefix that though as some face names might interfere with context (Rose, River, Jade).. sort of like the old m: prefix for metadata...  so...  _face:rose rose_ would look for photos of Rose holding a rose
- if a place name is entered, then addd that to the context. It would also need to be prefixed. It would also solve the problem of not being able to specify "search by state" in the dropdown.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7264

**Reply by iansewell** [reply]
thanks, sorry for the noise
